tests/integration-tests/src/test/resources/broker.properties

```diff
@@ -15,4 +15,6 @@
 # limitations under the License.
 #
 
-name=ConfiguredViaProperties
\ No newline at end of file
+# use a very innocuous bit of config b/c it will be picked up by all embedded servers in the integration tests
```

Comment:
If unit testing properly verified the file config is being picked up and used as expected I dont see a real need for the integration test using the actual default _broker.properties_ file, and especially not doing so for _every_ integration test even though the broker doesnt provide a default file. That just means the tests are _always_ going on a path that actual use typically wont use. Would seem nicer to use a non-default file or only create the file during the test that needs it.

Reply:
it is only the embedded broker server that knows about the default and I think that is the normal path for it, same as for broker.xml.

Reply:
The original change makes it look like the run command knows about it and checksthe default name, passing it on if it exists. The broker having and loading a file for _all_ the tests, that it is _not_ loading by default typically (as it doesnt exist) for normal users, means it isnt the normal path to me. It means the tests dont take the normal path.